### PR TITLE
Replace IRC with Slack

### PR DESCRIPTION
**Pull request by singingwolfboy**
@jbarciauskas @lamagnifica can you review this?

### CODE REVIEW

File: en_us/shared/preface.rst [hunk 1]
```diff
@@ -305,7 +305,8 @@ Additional Google groups are occasionally formed for individual projects.
  Please do not report security issues in public. If you have a concern,
  please email security@edx.org.
 
-EdX engineers often monitor the Freenode #edx-code IRC channel.
+EdX also runs a `Slack`_ team for real-time conversation about Open edX.
```
Comment (jbarciauskas) [inline en_us/shared/preface.rst]: Should we make both links take you to the invitation page?

**Comment by jbarciauskas** [comment]
1 minor comment and I don't know how the linkification magic works, but assuming it is right, :+1:

File: en_us/shared/preface.rst (hunk 1 above)
Comment (singingwolfboy) [inline en_us/shared/preface.rst]: Why? Not everyone knows what Slack is.

Comment (jbarciauskas) [inline en_us/shared/preface.rst]: Mainly on the principle that people are likely to only take away one action from any particular page or piece of documentation, and we'd like that to be joining our Slack community, not just learning about Slack

**Comment by lamagnifica** [comment]
@singingwolfboy please be sure to make the same edits to the preface.rst file in the edx-analytics-dashboard repo (as noted on line 7).
Also please be sure to follow the procedures outlined in https://openedx.atlassian.net/wiki/display/DOC/Doc+Team+Pull+Request+Template
for testing and followup

File: en_us/shared/preface.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/shared/preface.rst]: Per a11y best practices, do not link to the same destination multiple times in the same topic. 
To clarify what Slack is without a link, I suggest switching the order of the clauses (state a problem that readers might be trying to solve, then how to solve it). Maybe:  "For real-time conversations about Open edX, the edX open source community managers run a Slack team."

File: en_us/shared/preface.rst [hunk 2]
```diff
@@ -305,7 +305,8 @@ Additional Google groups are occasionally formed for individual projects.
...
+To join the team, `get a free invitation to Slack`_ and follow the instructions.
```
Comment (lamagnifica) [inline en_us/shared/preface.rst]: That landing page remains horrifying. How can we engage with whoever created it to correct the typo, and to add the obviously missing context of what this is and who might use it and why...?

Comment (jbarciauskas) [inline en_us/shared/preface.rst]: I fixed the typo. I think "horrifying" is a bit dramatic - generally people will have gotten that link from a source that contains the right context, no one will discover it on their own

Comment (lamagnifica) [inline en_us/shared/preface.rst]: (calling it like I experience it!) thank you very much for fixing it!

**Comment by lamagnifica** [comment]
I get two errors when testing this PR locally. A recently-merged PR will clear up those errors. 
@nedbat FYI

**Comment by lamagnifica** [comment]
Build errors are cleared up. Any further comment on this? @singingwolfboy do you plan to finish this up, or would you like someone else to complete it?

**Comment by singingwolfboy** [comment]
@lamagnifica I'll hand this off to you, if you're alright with that. Happy to help out as necessary, but you know this repo better than me.

**Pull request merged**